src/loaders/prowizard/eureka.c

```diff
@@ -163,6 +163,8 @@ static int test_eu(const uint8 *data, char *t, int s)
 			min_trk = trk;
 	}
 
+	PW_REQUEST_DATA(s, max_trk + (max_pat * 4 * 2) + 1085);
```

Comment:
This is wrong—the `trk` values are absolute offsets in the file, so this should just request `max_trk` bytes.

Reply:
If that is the only glitch with this changeset, and you patch it, I'll go ahead and merge this.

Reply:
That's the only bug I found with code inspection. I would still like to gather test files for the formats missing them just in case, but if I can't find some I'll go ahead and just patch this single bug.

Reply:
This should be fixed now.